src/foam/nanos/crunch/ui/MinMaxCapabilityWizardlet.js

```diff
@@ -163,7 +163,7 @@ foam.CLASS({
               data$: this.selectedData$,
               minSelected$: this.min$,
               maxSelected$: this.max$,
-              choiceView: { class: 'foam.u2.view.CardSelectView', largeCard: true }
+              choiceView: { class: this.capability.customCardSelectView, largeCard: true }
```

Comment:
Use the code from before passing in `'foam.u2.view.CardSelectView'`  and pass an `of` here grabbing class id from data. I would expect it to be NatureCode.

Pass it like how the `of` is passed here